### PR TITLE
Fix #36

### LINKED ISSUE #36: Auto complete after the "dot" doesn't replace the original text
See https://youtrack.jetbrains.com/issue/RIDER-97749?replyTo=27-8019313

When choosing an option here (after the dot):
image
This happens:
image

It doesn't happen when auto completing before the dot

### PR DESCRIPTION
**Pull request by Kuinox**
⚠️ There is no tests on this completion.  
Fix #36.

### CODE REVIEW

**Comment by ltrzesniewski** [comment]
Would you like to add a test, or is it not really necessary for this case?

**Comment by Kuinox** [comment]
I manually tested it, I made in sort the code shouldn't break stuff.  
I just hope it don't break stuff :D

File: Abc.MoqComplete/Abc.MoqComplete/CompletionProvider/SuggestMockProvider.cs [hunk 1]
```diff
@@ -50,7 +53,16 @@ protected override bool AddLookupItems(CSharpCodeCompletionContext context, IIte
...
-            lookupItem.WithInitializedRanges(context.CompletionRanges, context.BasicContext);
+
+            var node = context.NodeInFile;
+            while (!(node is ICSharpArgument) && node != null) node = node?.Parent;
+            if (node != null)
+            {
+                var arg = (ICSharpArgument)node;
+                var range = arg.GetExtendedDocumentRange();
+                lookupItem.SetRanges(context.CompletionRanges.WithReplaceRange(range));
+            }
```
Comment (ltrzesniewski) [inline Abc.MoqComplete/Abc.MoqComplete/CompletionProvider/SuggestMockProvider.cs:64]: Should we add something like `else { lookupItem.WithInitializedRanges(context.CompletionRanges, context.BasicContext); }` here?

Comment (Kuinox) [reply Abc.MoqComplete/Abc.MoqComplete/CompletionProvider/SuggestMockProvider.cs:64]: Why not. The initialized ranges doesn't work for some reason here and always return an empty range, but it wouldn't hurt to do that.

**Pull request merged**